client/me/profile/index.jsx

```diff
@@ -121,18 +127,31 @@ class Profile extends Component {
 							/>
 						</FormFieldset>
 
-						<FormFieldset
-							className={ clsx( {
-								'profile__gravatar-fieldset-is-loading': this.props.isFetchingUserSettings,
-							} ) }
-						>
-							<ToggleControl
-								disabled={ this.props.getDisabledState() || this.props.isFetchingUserSettings }
-								checked={ this.props.getSetting( 'gravatar_profile_hidden' ) }
-								onChange={ this.toggleGravatarHidden }
-								label={ <UpdatedGravatarString gravatarProfileLink={ gravatarProfileLink } /> }
-							/>
-						</FormFieldset>
+						<p className="profile__gravatar-profile-description">
+							<span>
+								{ this.props.translate(
+									'Your profile is powered by Gravatar. Your Gravatar is public by default and may appear on any site using Gravatar when you’re logged in with {{strong}}%(email)s{{/strong}}.' +
+										' To manage your profile and privacy settings, {{button}}visit your Gravatar profile here{{/button}}.',
+									{
+										components: {
+											strong: <strong />,
+											button: (
+												<button
+													className="profile__gravatar-profile-button"
+													onClick={ this.openGravatarQuickEditor }
+												/>
```

Comment:
I'm a bit concerned about the decision to use the button here - is it necessary? An ordinary link seems more appropriate and would save you a bit of styling. It would also avoid the current issue with how the text breaks weirdly.

<img width="191" alt="Screenshot 2024-07-29 at 12 13 19" src="https://github.com/user-attachments/assets/2798469b-3422-4b50-b770-2cb06065a147">

If a button is necessary, you could use the `Button` component and the `link` prop to also avoid the additional styles.

Reply:
@Aurorum Thank you for pointing out the issue, it has been fixed through [this commit](https://github.com/Automattic/wp-calypso/pull/93023/commits/63485738683e8cc9d6430f7fe24ee8c7ffac6ebf).